### ISSUE TITLE
Missing "git rebase --continue"

### ISSUE THREAD

**Issue by thomasroefer** [brotherbard/gitx#59]
If rebasing fails, it would be nice to be able to continue it after the conflicts have been resolved. Maybe I have overlooked something, but I have not found a way to continue rebasing.

**Comment by brotherbard** [comment]
No, GitX currently does not support continuing rebase or merge. That would be a good thing to have though.
